diff --git a/src/MEGASync/CMakeLists.txt b/src/MEGASync/CMakeLists.txt
index bba820eea..bf859f263 100644
--- a/src/MEGASync/CMakeLists.txt
+++ b/src/MEGASync/CMakeLists.txt
@@ -59,6 +59,8 @@ elseif (APPLE)
     )
 endif()
 
+include_directories( ${CMAKE_CURRENT_SOURCE_DIR} )
+
 target_sources(MEGAsync
     PRIVATE
     ${MEGA_DESKTOP_APP_HEADERS}
diff --git a/src/MEGASync/MegaApplication.cpp b/src/MEGASync/MegaApplication.cpp
index af1a99138..ed1131675 100644
--- a/src/MEGASync/MegaApplication.cpp
+++ b/src/MEGASync/MegaApplication.cpp
@@ -4,7 +4,7 @@
 #include "Avatar.h"
 #include "ChangeLogDialog.h"
 #include "CommonMessages.h"
-#include "CrashHandler.h"
+//#include "CrashHandler.h"
 #include "CrashReportDialog.h"
 #include "DateTimeFormatter.h"
 #include "DialogOpener.h"
diff --git a/src/MEGASync/control/control.cmake b/src/MEGASync/control/control.cmake
index 8d6792428..89facb674 100644
--- a/src/MEGASync/control/control.cmake
+++ b/src/MEGASync/control/control.cmake
@@ -4,7 +4,7 @@ set(DESKTOP_APP_CONTROL_HEADERS
     ${CMAKE_CURRENT_LIST_DIR}/AppStatsEvents.h
     ${CMAKE_CURRENT_LIST_DIR}/AsyncHandler.h
     ${CMAKE_CURRENT_LIST_DIR}/ConnectivityChecker.h
-    ${CMAKE_CURRENT_LIST_DIR}/CrashHandler.h
+    #${CMAKE_CURRENT_LIST_DIR}/CrashHandler.h
     ${CMAKE_CURRENT_LIST_DIR}/DialogOpener.h
     ${CMAKE_CURRENT_LIST_DIR}/DownloadQueueController.h
     ${CMAKE_CURRENT_LIST_DIR}/EmailRequester.h
@@ -57,7 +57,7 @@ set(DESKTOP_APP_CONTROL_SOURCES
     ${CMAKE_CURRENT_LIST_DIR}/AppState.cpp
     ${CMAKE_CURRENT_LIST_DIR}/AppStatsEvents.cpp
     ${CMAKE_CURRENT_LIST_DIR}/ConnectivityChecker.cpp
-    ${CMAKE_CURRENT_LIST_DIR}/CrashHandler.cpp
+    #${CMAKE_CURRENT_LIST_DIR}/CrashHandler.cpp
     ${CMAKE_CURRENT_LIST_DIR}/DialogOpener.cpp
     ${CMAKE_CURRENT_LIST_DIR}/DownloadQueueController.cpp
     ${CMAKE_CURRENT_LIST_DIR}/EmailRequester.cpp
diff --git a/src/MEGASync/main.cpp b/src/MEGASync/main.cpp
index 6832501a4..cd03e9d52 100644
--- a/src/MEGASync/main.cpp
+++ b/src/MEGASync/main.cpp
@@ -1,5 +1,5 @@
 #ifndef DEBUG
-#include "CrashHandler.h"
+//#include "CrashHandler.h"
 #endif
 
 #include "MegaApplication.h"
diff --git a/src/MEGAUpdateGenerator/CMakeLists.txt b/src/MEGAUpdateGenerator/CMakeLists.txt
index ce4747fce..45f0a0a91 100644
--- a/src/MEGAUpdateGenerator/CMakeLists.txt
+++ b/src/MEGAUpdateGenerator/CMakeLists.txt
@@ -15,10 +15,11 @@ target_sources(MEGAUpdateGenerator
 )
 
 # Load and link needed libraries for the CHATlib target
-find_package(cryptopp CONFIG REQUIRED)
+add_library (cryptopp SHARED IMPORTED)
+set_target_properties(cryptopp PROPERTIES IMPORTED_LOCATION /app/lib/libcryptopp.so)
 target_link_libraries(MEGAUpdateGenerator
     PRIVATE
-    cryptopp::cryptopp
+    cryptopp
     MEGA::SDKlib
 )
 
diff --git a/cmake/modules/get_clang_format.cmake b/cmake/modules/get_clang_format.cmake
index 69b43d5..9ad26a4 100644
--- a/cmake/modules/get_clang_format.cmake
+++ b/cmake/modules/get_clang_format.cmake
@@ -1,14 +1,2 @@
 function(get_clang_format)
-    message(STATUS "Downloading .clang-format")
-
-    set(FILE_URL "https://raw.githubusercontent.com/meganz/sdk/master/.clang-format")
-    set(OUTPUT_FILE ${PROJECT_SOURCE_DIR}/.clang-format)
-    file(DOWNLOAD ${FILE_URL} ${OUTPUT_FILE}
-        SHOW_PROGRESS
-        STATUS status
-    )
-
-    if(NOT status EQUAL 0)
-        message(FATAL_ERROR "Download failed with status: ${status}")
-    endif()
 endfunction()

