diff --git a/src/MEGASync/mega/src/gfx/freeimage.cpp b/src/MEGASync/mega/src/gfx/freeimage.cpp
index 2f685279a..a6883ed4e 100644
--- a/src/MEGASync/mega/src/gfx/freeimage.cpp
+++ b/src/MEGASync/mega/src/gfx/freeimage.cpp
@@ -251,7 +251,7 @@ ScopeGuard<F, P> makeScopeGuard(F f, P p){ return ScopeGuard<F, P>(f, p);	}
 bool GfxProviderFreeImage::readbitmapFfmpeg(FileSystemAccess* fa, const LocalPath& imagePath, int size)
 {
 #ifndef DEBUG
-    av_log_set_level(AV_LOG_PANIC);
+    //av_log_set_level(AV_LOG_PANIC);
 #endif
 
     // Open video file
